docs/content/contributing/overview/index.md

```diff
@@ -0,0 +1,43 @@
+---
+type: docs
+title: "Overview: Contributing to Radius"
+linkTitle: "Overview"
+description: "Guides and requirements for contributing to Radius"
+weight: 100
+---
+
+We welcome contributions to Radius! Contributions can come in different ways such as engaging with the community, contributing code, or improving the documentation. This page provides an overview of the different ways you can contribute to Radius.
+
+## Community
+
+Check out the [Radius Community]({{< ref community >}}) page to learn about the different ways you can engage with the Radius community.
+
+## GitHub
+
+If you would like to file Issues, access the source code, or use Codespaces please visit the [Radius GitHub repo](https://github.com/radius-project). 
+
+### good-first-issues
+
+To quickly get started with contributing to code on Radius, here are some identified [good-first-issues](https://aka.ms/radius-first-issues) that you can `/assign` to yourself and start contributing. 
```

Comment:
For links to outside pages, such as GitHub in this case, could you add a button below the section please?